### DISCUSSION TITLE
Add beatmap progress to multi listing and lobby screen

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#14898] [Ideas] [answered]
It would be really nice to know how far into the beatmap the room is when choosing what room to join. In stable it happens all the time that peole join a started room, wait 1 minute and then leave because they are tired of waiting because someone decided to play a 6 minutes map. Next to the beatmap name that was added earlier we could have a 1:20 / 3:20 watch as a temporary nice QoL addition.

**Comment by ghost** [comment]
Sounds simple & effective. Would really save a lot of time/pain.

**Comment by integerrr** [comment, marked as answer]
For documentation's sake: #15172

**Reply by peppy** [reply]
You can find issues to the right.

![20211019 092134 (Safari)](https://user-images.githubusercontent.com/191335/137823818-452d589d-66ea-4dd1-971b-00450c75f56b.png)
